CK3toEU4/Source/Mappers/TitleTagMapper/TitleTagMapping.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TITLE_TAG_MAPPING_H
+#define TITLE_TAG_MAPPING_H
+
+#include "Parser.h"
+#include <set>
+
+namespace mappers
+{
+class TitleTagMapping: commonItems::parser
+{
+  public:
+	TitleTagMapping() = default;
+	explicit TitleTagMapping(std::istream& theStream);
+
+	[[nodiscard]] std::optional<std::string> titleMatch(const std::string& ck3title) const;
+	[[nodiscard]] std::optional<std::string> capitalMatch(int eu4capital) const;
+	[[nodiscard]] bool fallbackMatch() const;
+
+	[[nodiscard]] const auto& getEU4Tag() const { return eu4Tag; }		 // used for testing
```

Comment:
You should be able to drop these and the associated tests. You're testing the functionality you actually care about.